### ISSUE TITLE
Can I run it without JCE?

### ISSUE THREAD

**Issue by georgy7** [konachan700/JNekoImageDB#12]
I do not need encryption.

Can I completely disable the encryption feature of a database from the very beginning?

I don't need the network features either (P2P, comments, likes/dislikes, VK, cloud integration).

**Comment by konachan700** [comment]
Yes. You can disable encryption in config. JCE is not needed by default, because AES128 used.
Networks features is not implemented at this moment.

**Comment by georgy7** [comment]
Is there any documentation about the config file. Is this a file?
